Wrappers/Python/cil/plugins/ccpi_regularisation/functions/regularisers.py

```diff
@@ -358,4 +462,5 @@ def check_input(self, input):
         DataOrder.check_order_for_engine('cil', input.geometry)
         if ( input.geometry.channels == 1 ) or ( not input.geometry.length == 3) :
             raise ValueError('TNV requires 2D+channel data. Got {}'.format(input.geometry.dimension_labels))
-        
\ No newline at end of file
+        
```

Comment:
Following the previous comment: this needs

```python

if self.strong_convexity_constant>0:
    in_arr *= (1 + tau*self.strong_convexity_constant)
```

Reply:
Fixed